src/web/src/components/Posts/ExpandIcon.tsx

```diff
@@ -0,0 +1,52 @@
+import { IconButton, createStyles } from '@material-ui/core';
+import { makeStyles } from '@material-ui/core/styles';
+import ExpandMoreIcon from '@material-ui/icons/ExpandMore';
+import ExpandLessIcon from '@material-ui/icons/ExpandLess';
+
+type Props = {
+  small: Boolean;
+  expandHeader: Boolean;
+};
+
+const useStyles = makeStyles(() =>
+  createStyles({
+    container: {
+      display: 'flex',
+      justifyContent: 'center',
+    },
+    smallIcon: {
+      padding: 0,
+      fill: '#cccccc',
+    },
+    bigIcon: {
+      fontSize: '5rem',
+      fill: '#cccccc',
+    },
+    iconBtn: {
+      padding: 0,
+    },
+  })
+);
+const ExpandIcon = ({ small, expandHeader }: Props) => {
+  const classes = useStyles();
+  return small ? (
+    <div className={classes.container}>
+      <IconButton className={classes.iconBtn}>
+        {expandHeader ? (
+          <ExpandLessIcon className={classes.bigIcon} />
+        ) : (
+          <ExpandMoreIcon className={classes.bigIcon} />
+        )}
+      </IconButton>
+    </div>
+  ) : (
+    <IconButton className={classes.iconBtn}>
+      {expandHeader ? (
+        <ExpandLessIcon className={classes.smallIcon} />
+      ) : (
+        <ExpandMoreIcon className={classes.smallIcon} />
```

Comment:
The expand more icon doesn't show up when we have the info section collapse, so we're only seeing it when we close it and see a quick flash of it
![chrome_bjEnvfSajU](https://user-images.githubusercontent.com/77639637/151067111-9f491145-9cc6-4e43-91c4-dc3178abeb9a.gif)

Can you decide if we need it and use it, or remove the code for it please

Reply:
I'll remove the condition that toggles the big icon. Maybe we don't need it